data/demo/results_for_checking-spliced/SEND_Log.md

```diff
@@ -0,0 +1,19 @@
+__There are no leavers from AY 20__
```

Comment:
Are you happy with these warnings?

Reply:
Yeah, no issues around these. (The previous data had the same ones plus a lot more: https://github.com/MastodonC/witan.send/blob/master/data/demo/results/SEND_Log.md)